apps/frontend/src/app/5_pages/BitocracyPage/components/ProposalCardsMobile/ProposalCardsMobile.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC, useCallback } from 'react';
+
+import { t } from 'i18next';
+import { useNavigate } from 'react-router-dom';
+
+import { Icon, IconNames, Paragraph } from '@sovryn/ui';
+
+import { translations } from '../../../../../locales/i18n';
+import { Proposal } from '../../../../../utils/graphql/rsk/generated';
+import { renderEndDate } from '../../BitocracyPage.utils';
+import { ProposalStatus } from '../ProposalStatus/ProposalStatus';
+import { ProposalType } from '../ProposalType/ProposalType';
+
+type ProposalCardsMobileProps = {
+  proposals: Proposal[];
+  isLoading: boolean;
+};
+
+export const ProposalCardsMobile: FC<ProposalCardsMobileProps> = ({
+  proposals,
+  isLoading,
+}) => {
+  const navigate = useNavigate();
+
+  const handleCardClick = useCallback(
+    (proposal: Proposal) => navigate(`/bitocracy/${proposal.proposalId}`),
+    [navigate],
+  );
+
+  const renderProposals = () => {
+    if (!proposals || proposals.length === 0) {
+      return (
+        <>
+          {isLoading ? (
+            Array.from(Array(4).keys()).map(i => (
+              <div key={i} className="relative text-center my-1">
+                <span className="h-10 block bg-gray-50 rounded animate-pulse" />
+              </div>
+            ))
+          ) : (
+            <div className="text-center italic my-1">
+              {t(translations.common.tables.noData)}
+            </div>
+          )}
+        </>
+      );
+    }
+
+    return proposals.map(proposal => (
+      <div
+        key={proposal.id}
+        className="rounded p-4 bg-gray-70 mb-2 relative"
+        onClick={() => handleCardClick(proposal)}
```

Comment:
Not sure clicking anywhere on the proposal is good idea. Not very good accessibility and user experience if I want to copy / scroll or etc - as it may open proposal page instead of what I wanted to actually to do. @john-light

Reply:
In my opinion everything that should open or redirect to page should look or be a hyperlink (it also helps for SEO)